doc/source/examples/extended_examples/executable/executable.rst

```diff
@@ -101,6 +101,64 @@ This code shows how you can input other arguments:
    The first natural frequency is 344.28 Hz.
 
 
+Convert the app to an executable
+================================
+
+Using the Python library `PyInstaller <https://pyinstaller.org>`, it is possible to convert the app to an executable by taking a few precautions.
+
+.. code:: console
+
+   pyi-makespec cli_rotor.py
+
+Add the argument ``--onefile`` to put everything in a single file. Otherwise you will have a folder containing the application and dll dependencies.
+
+.. code:: console
+
+   pyi-makespec cli_rotor.py --onefile
+
+After the first line in the file ``cli_rotor.spec``, add the following lines:
+
+.. code:: python
+
+   import os
+   import importlib
+
+   proot = os.path.dirname(importlib.import_module("ansys.api.mapdl").__file__)
+   files_to_add = [
+       (os.path.join(proot, "VERSION"), ".\\ansys\\api\\mapdl")
+   ]  # use / instead of \\ if you are on Linux
+
+Add ``files_to_add`` to ``Analysis`` data
+
+.. code-block:: python
+   :emphasize-lines: 5
+
+   a = Analysis(
+       ["cli_rotor.py"],
+       pathex=[],
+       binaries=[],
+       datas=files_to_add,
+       hiddenimports=[],
+       hookspath=[],
+       hooksconfig={},
+       runtime_hooks=[],
+       excludes=[],
+       win_no_prefer_redirects=False,
+       win_private_assemblies=False,
+       cipher=block_cipher,
+       noarchive=False,
+   )
+
+Then generate the executable form the .spec file
```

Comment:
Please add also the following link: https://pyinstaller.org/en/stable/spec-files.html

Reply:
I've added a link [more earlier](https://github.com/ansys/pymapdl/pull/2100/files#diff-32d8dfd777a43e2e0eb0d74e81e2f7a550c314abd47a3476905733c8c92803e9R110) in the example.
Is it good ?

Reply:
Yes. It is good.